### DISCUSSION TITLE
Disk image loses everything after restart VM

### DISCUSSION THREAD

**Discussion by CharlieAndLouise** [utmapp/UTM#3742] [UTM Help (macOS)]
I installed Debian or Ubuntu desktop in UTM. Then after rebooting I can login Debian. Next I clicked 'shuts down and stops the VM' and the VM window is closed. 

Now the issue is I tried to start the VM again. The disk image looks new and the boot option - 'debian' or 'ubantu' is missing in bios setting.

I did some extra try -
shut down the VM in Debian - Safe
use the triangle on title bar (restarts the VM) - Safe

* UTM Version:  3.0.4
* OS Version: 
* Intel or Apple Silicon? Apple M1

**Comment by osy** [comment]
You may not have installed GRUB boot manager correctly.

**Reply by CharlieAndLouise** [reply]
thanks osy, could you please give me a step to step installation menu? Because I didn't see this part in any videos or installation menu.
